Unity Example/Assets/Discord RPC/Scripts/Presence/DiscordPresence.cs

```diff
@@ -98,11 +106,33 @@ public DiscordPresence(DiscordRPC.BaseRichPresence presence)
 				this.largeAsset = new DiscordAsset();
 			}
 
+			if (presence.HasButtons())
+			{
+				this.buttons = new DiscordButton[]
+				{
+					new DiscordButton()
+					{
+						label = presence.Buttons[0].Label,
+						url = presence.Buttons[0].Url,
+					},
+					new DiscordButton()
+					{
+						label = presence.Buttons[1].Label,
+						url = presence.Buttons[1].Url,
+					}
+				};
```

Comment:
This shouldn't be a new array of only 2 items. Copy the entire presence.Buttons object.

Buttons is an array with an **unbounded length**. If this is incorrect, please create a PR to fix this in the base library. The Unity package should match functionality of the base library.

Reply:
fixed